contents/blog/ceo-diary-1.md

```diff
@@ -0,0 +1,71 @@
+---
+date: 2021-08-19
+title: Winning from the back - late mover advantage
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ceo-diaries
+author: james-hawkins
+---
+
+We are open source is [literally our top value](../handbook/company/values)... and what better way to be transparent than to share a diary!
+
+# Late mover advantage
+
+Product analytics is a busy space. It reminds me of the _front_ entrance to Liverpool Street station in London:
+
+![Liverpool Street Front Entrance - lots of people](../images/blog/CEO-diary-1/liverpool-st-station.jpg)
+
+PostHog Fact: Tim (my cofounder) incorrectly believes this is the side entrance - it's one of our greatest disagreements.
+
+There are tons of competitors in product analytics, and they've pretty much all got 300+ person teams, around $100M or more in funding, and years and years of headstart. Yet, we're totally confident we can win as a late mover.
+
+A losing strategy for late movers:
+
+- We'll hire lots of salespeople
+- We'll build features that the salespeople need to sell to big companies, which will make our product feel the same as everyone else's
+- Since the product is built for selling and not using, it'll be annoying to use
+- We will churn lots of customers, and thus will put even more energy into selling. Go back to step 1.
+
+A winning strategy for late movers:
+
+- We'll double down investing in our open source project
+- It gets more and more popular so developers at big companies will find and deploy us
+- Since the product is good, it'll roll out across the rest of the company
+- Big companies end up happily paying for the valuable product they're using. Go back to step 1.
+
+# So what are we doing about the above?
+
+PostHog started off as an open source project. We started getting emails like this one, which is from a 30k employee corporate that you'll have heard of:
+
+![An email from a potential customer saying they have extreme interest](../images/blog/CEO-diary-1/extreme-interest.jpg)
```

Comment:
Nit: this could be a quote.

Reply:
I kind of like screenshotting an email as it makes it feel a bit more real